packages/variable-editor/src/VariableEditor.tsx

```diff
@@ -97,6 +103,7 @@ function VariableEditor(props: EditorProps) {
           setVariables={setVariables}
           setSelectedVariablePath={setSelectedVariablePath}
           validationMessages={validationMessages}
+          knownVariables={knownVariables}
```

Comment:
why do you request fetch this data here and not where it is needed?
This approach has 2 disadvantaches:
- The data is initially loaded for everyone which opens this editor even if he doesn't open this browser at all.
- You need to add properties to the VariablesMaster component, which blows up the interface itself